superdesk/text_checkers/spellcheckers/default.py

```diff
@@ -0,0 +1,57 @@
+# -*- coding: utf-8; -*-
+#
+# This file is part of Superdesk.
+#
+# Copyright 2013-2019 Sourcefabric z.u. and contributors.
+#
+# For the full copyright and license information, please see the
+# AUTHORS and LICENSE files distributed with this source code, or
+# at https://www.sourcefabric.org/superdesk/license
+
+import re
+import logging
+# from superdesk.errors import SuperdeskApiError
+import superdesk
+from superdesk.text_checkers.spellcheckers import SPELLCHECKER_DEFAULT
+from superdesk.text_checkers.spellcheckers.base import SpellcheckerBase
+
+logger = logging.getLogger(__name__)
+
+
+class Default(SpellcheckerBase):
+    """Default spellchecker of Superdesk
+
+    This spellchecker works with internal dictionaries, in any language.
+    """
+
+    name = SPELLCHECKER_DEFAULT
+    label = "Superdesk default spellchecker"
+    capacities = ["spelling"]
+    languages = ['*']
```

Comment:
also this could be a constant `SpellcheckerBase.ANY_LANGUAGE``

Reply:
right